lms/services/application_instance.py

```diff
@@ -316,6 +316,10 @@ def update_from_lti_params(self, application_instance, params: LTIParams):
         ]:
             setattr(application_instance, attr, params.get(attr))
 
+        # This is a potentially misleading, as we can get here from deep-linked
+        # "launches". Depending on whether you count that as a launch or not.
+        application_instance.last_launched = datetime.now()
```

Comment:
I think counting the deep linking is correct, is "technically" also a launch and it shows activity on the AI.

In the events table we have the raw data if that distinction mattered for some troubleshooting.